docs/sections/Docs-Labels/index.jsx

```diff
@@ -4,7 +4,98 @@ export default class Labels extends React.Component {
   render() {
     return (
       <div>
-      </div>
+      	 <h2 className="title">
```

Comment:
This is specific to the docs, you can move it to `docs/styles/main.scss` or just leave it alone (aka, delete changes related to doc styling).